pkg/releasing/git.bzl

```diff
@@ -89,7 +89,7 @@ def git_changelog(name, **kwargs):
         name = name,
         # This requires bazel 4.x
         target_compatible_with = select({
-            "//toolchains/git:have_git": [],
+            "@rules_pkg//toolchains/git:have_git": [],
```

Comment:
Won't this break people who rename the repo to a new name on import?
Have you tried doing that?

Reply:
Pretty sure it will break. It looks like a flaw in Bazel tbh. I thought that all the refs/labels should be resolved within the repo - which would be the external (imported) repo. But apparently it's not. It tries to resolve those from within the host workspace, which seem ood.

Reply:
@@ -89,7 +91,7 @@ def git_changelog(name, **kwargs):
         name = name,
         # This requires bazel 4.x
         target_compatible_with = select({
-            "//toolchains/git:have_git": [],
+            str(Label("//toolchains/git:have_git")): [],
             "//conditions:default": ["//:not_compatible"],
         }),